src/main/java/org/minions/devfund/lourdes/Movie.java

```diff
@@ -0,0 +1,85 @@
+package org.minions.devfund.lourdes;
+
+import java.util.ArrayList;
+
+/**
+ * Class that define the Movie entity.
+ */
+public class Movie {
+    private String name;
+    private ArrayList<Actor> actors;
+    private double rating;
+
+    /**
+     * Constructor that receive the Movie Name.
+     *
+     * @param name represents the movie name.
+     */
+    public Movie(final String name) {
+        this.name = name;
+        this.actors = new ArrayList<Actor>();
+    }
+
+    /**
+     * Constructor.
+     */
+    public Movie() {
+        this("");
+    }
+
+    /**
+     * Method that get the name of the movie.
+     *
+     * @return the movie name.
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Method that set the movie name.
+     *
+     * @param name movie name.
+     */
+    public void setName(final String name) {
+        this.name = name;
+    }
+
+    /**
+     * Method that get the actor list.
+     *
+     * @return the actor list.
+     */
+    public ArrayList<Actor> getActors() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)